libs/wingsdk/test/global.setup.ts

```diff
@@ -0,0 +1,6 @@
+import { exec, execSync } from "child_process";
+
+export default async function () {
+  execSync("npx tsc -p tsconfig.test.json");
```

Comment:
```suggestion
  // compile src/**/*.on*.inflight.ts to .js because these are going to be
  // injected into our javascript vm and cannot be resolved via vitest
  execSync("npx tsc -p tsconfig.test.json");
```